### DISCUSSION TITLE
problem with update

### DISCUSSION THREAD

**Discussion by ottl05** [rern/rAudio#951] [General]
Hi,
I´m running raudio v 20220410 and want to update to  20220618, but I´m getting the message:
" please download the latest image file to be able to update"

what should I do?

thanks

**Comment by EarsofTin** [comment]
See discussion #620 for a solution that avoids reimaging  your SD card:
https://github.com/rern/rAudio-1/discussions/620

**Comment by ottl05** [comment]
thanks, I changed the date to 20220518 and it worked :-)

**Comment by rern** [comment]
Bypassing will be fine as long as those updated files beyond last 30 days are not used. 
Should there be any issues, try the latest image file before reporting a bug.

**Comment by digixmax** [comment]
FWIW - I tried the same fix: I changed the date in the file /srv/http/data/addons/r1 to 20220531, after which I was able to execute the update in AddOn, but then the system appears to hang on reboot: the web page at its network IP address would not come up.
